Sources_v3/Database/StreamChatModel.xcdatamodeld/StreamChatModel.xcdatamodel/contents

```diff
@@ -83,6 +84,11 @@
             </uniquenessConstraint>
         </uniquenessConstraints>
     </entity>
+    <entity name="DeviceDTO" representedClassName="DeviceDTO" syncable="YES">
```

Comment:
Shouldn't we mark `id` as a uniqueness constraint?

Reply:
Added in 107624943165ad39afd991a7fd9aba8d58c1a28a